code/modules/mob/living/update_status.dm

```diff
@@ -78,30 +78,24 @@
 		return TRUE
 
 //Updates canmove, lying and icons. Could perhaps do with a rename but I can't think of anything to describe it.
-/mob/living/update_canmove(delay_action_updates = 0)
-	var/fall_over = !can_stand()
-	var/buckle_lying = !(buckled && !buckled.buckle_lying)
-	if(fall_over || resting || IsStunned() || (buckled && buckle_lying != 0))
+/mob/living/update_canmove (delay_action_updates = 0)
```

Comment:
```suggestion
/mob/living/update_canmove(delay_action_updates = 0)
```
Not sure this is purposeful or not, just want to point it out incase

Reply:
I plan on nuking this proc entirely eventually.